webapp/sources/rudder/rudder-web/src/main/elm/rules/sources/ViewRuleDetails.elm

```diff
@@ -98,6 +98,10 @@ editionTemplate model details =
     nbInclude = case Maybe.Extra.unwrap [] .targets originRule of
       [Composition (Or i) (Or e)] -> List.length i
       targets -> List.length targets
+
+    nbDirectives = case originRule of
+      Just oR -> String.fromInt (List.length oR.directives)
```

Comment:
Why the number from the original Rule and not for the current Rule ? does not that number need to change if you add directives ? Same question for group though

Reply:
Because I think we should display the number of directives/groups that are actually applied to the rule, and this should not change until we save our changes. We keep track of the number of directives we've added / removed in another badge next to it.

I could be wrong though.

Reply:
Ha yes totally !